lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
I'd prefer to use `deprecateUI` so all writes go through the project's `ui` object (to honor things like `--silent` flag and whatnot).

https://github.com/ember-cli/ember-cli/blob/master/lib/utilities/deprecate.js#L11